.github/scripts/check-via-duplication.py

```
@@ -0,0 +1,107 @@
+#!/usr/bin/env python3
+import argparse, json, re, subprocess, sys, tempfile
+from pathlib import Path, PurePosixPath
+
+ROOT = Path(__file__).resolve().parents[2]; VERSION = "cpd 5.0.12"
```

Ruff (0.15.21)

[error] Multiple statements on one line (semicolon)

(E702)